grails-app/domain/org/pih/warehouse/order/Order.groovy

```diff
@@ -456,8 +457,37 @@ class Order implements Serializable {
         return !hasInvoice && isPrepaymentRequired && hasActiveItemsOrAdjustments
     }
 
+    /**
+        Should return true if at least one PO item is invoiceable.
+        A PO item is invoiceable if the order has a prepayment invoice and at least one of those requirements are fulfilled:
+            1. There is an order adjustment on that item that hasn’t already been invoiced
```

Comment:
I think the first point is misleading. 

> What is an invoiceable item?
a shipped item or a placed adjustment; any placed adjustment is pulled as invoiceable on the partial final invoice generated (but it can be removed or edited) or a canceled line

**Order adjustment** should be invoiceable if the order has a prepayment invoice and is placed.
**Order item** should be invoiceable if it has prepayment invoice and:
1. is partially shipped (only this partial quantity not yet invoiced is invoiceable)
2. items canceled that are not yet invoiced

Imho there is no need to check in order item is invoiceable if it has `NotInvoicedAdjustment`, I think the order adjustments should be separated from order items and checked as it is written above.

(to be confirmed)